lib/data-types.js

```diff
@@ -789,14 +790,38 @@ UUID.prototype.validate = function validate(value, options) {
   return true;
 };
 
+/**
+ * A column storing a modified uuidv1 as binary(16).
+ * @see https://www.percona.com/blog/2014/12/19/store-uuid-optimized-way/
+ *
+ * @namespace DataTypes.BINARYUUID
+ */
+function BINARYUUID() {
```

Comment:
@sushantdhiman I was wondering if you want to enforce these to be classes (and be called with `new`) in v5?

Reply:
I'd prefer it!  Was just following what I saw previous!